tools/Dockerfile

```diff
@@ -7,13 +7,13 @@ RUN dnf copr enable project-flotta/flotta -y
 # Yum dependencies
 RUN dnf update -y \
   && dnf install -y openssl procps-ng dmidecode nc iproute \
-  yggdrasil flotta-agent-race node_exporter
+  yggdrasil flotta-agent-race node_exporter systemd-container
 
 # Modify podman configuration
 RUN sed -i s/netns=\"host\"/netns=\"private\"/g /etc/containers/containers.conf && \
     sed -i s/utsns=\"host\"/utsns=\"private\"/g /etc/containers/containers.conf
 
-RUN podman pull quay.io/bitnami/nginx:1.21.6
+#RUN podman pull quay.io/bitnami/nginx:1.21.6
```

Comment:
why to comment? should we remove?

Reply:
I guess that it's because each user will have his own images, no?

Reply:
That's leftover, sorry